cli/modelsvalidator/validate.py

```diff
@@ -0,0 +1,369 @@
+import json
+import re
+import sys
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import yaml
+from datastore.shared.util.key_strings import KEYSEPARATOR
+from datastore.shared.util.key_types import (
+    collection_regex,
+    collectionfield_regex,
+    field_regex,
+)
+
+DEFAULT_FILES = [
+    "./global/meta/models.yml",
+]
+
+decimal_regex = re.compile("^\d+\.\d{6}$")  # NOQA
```

Comment:
Defined in https://github.com/OpenSlides/openslides-backend/blob/d06365386e2b39eb001c2d56f2ea6e940099d61d/openslides_backend/shared/schema.py#L29 (slightly different even! That's why code duplication is bad ;)), please unify

Reply:
It's not only different, one of them is wrong.
The one from validate.py allows only positive decimals in the models.yml.
The other one allows also negative numbers in the schema.